### PR TITLE
Improve weekly test stability

### PR DESCRIPTION
**Pull request by jianyangli**
Address #335

### CODE REVIEW

**Comment by jianyangli** [comment]
Looks like this PR should wait until after #338 is merged.  Some test failures due to the use of `@remote_data`, which will be fixed by #338

**Comment by jianyangli** [comment]
@mkelley I only changed 5 files instead of 9 as reported by the PR because rebase included those you changed in PR #338 .  FYI for your review.  Thanks.

**Review by mkelley — APPROVED** [review]

**Pull request merged**
